wire-ios-request-strategy/Sources/Request Strategies/User Clients/MissingClientsRequestStrategy.swift

```diff
@@ -1,203 +0,0 @@
-//
-// Wire
-// Copyright (C) 2016 Wire Swiss GmbH
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see http://www.gnu.org/licenses/.
-//
-
-import Foundation
-
-/// Register new client, update it with new keys, deletes clients.
-@objc
-public final class MissingClientsRequestStrategy: AbstractRequestStrategy, ZMUpstreamTranscoder, ZMContextChangeTrackerSource {
```

Comment:
I'm deleting this strategy now since It's redundant now that send all messages using the `MessageSender`. The `MissingClientsRequestStrategy` was observing the `missingClients` relationship on the self `UserClient`, and would attempt to establish a session with all missing clients. 

The old message sending code path would wait for the the set of missing clients to become empty before attempting to encrypt and send a proteus message. The new async code path for message sending fetches missing clients using an async function and thus doesn't need to this state.